GTG/gtk/browser/taskbrowser.ui

```diff
@@ -8,529 +8,176 @@
-                    <property name="can_focus">False</property>
-                    <property name="label" translatable="yes">Dismiss</property>
-                    <property name="stock_id">gtk-close</property>
-                    <signal name="clicked" handler="on_dismiss_task" swapped="no"/>
+                    <property name="label" translatable="yes">Actionable Tasks</property>
+                    <property name="tooltip_text" translatable="yes">aka WorkView</property>
+                    <signal name="clicked" handler="temp_workview" swapped="no"/>
+                    <property name="valign">center</property>
                   </object>
-                  <packing>
-                    <property name="expand">False</property>
-                    <property name="homogeneous">True</property>
-                  </packing>
-                </child>
-                <child>
-                  <object class="GtkToolButton" id="delete_b">
-                    <property name="sensitive">False</property>
-                    <property name="can_focus">False</property>
-                    <property name="label" translatable="yes">Delete</property>
-                    <property name="icon_name">edit-delete</property>
-                    <signal name="clicked" handler="on_delete_task" swapped="no"/>
-                  </object>
-                  <packing>
-                    <property name="expand">False</property>
-                    <property name="homogeneous">True</property>
-                  </packing>
+                  <style>
+                    <class name="text-button"/>
+                  </style>
                 </child>
                 <child>
-                  <object class="GtkSeparatorToolItem" id="separator_7">
+                  <object class="GtkToggleButton" id="closed_tasks">
                     <property name="visible">True</property>
-                    <property name="can_focus">False</property>
+                    <property name="label" translatable="yes">Closed Tasks</property>
+                    <property name="tooltip_text" translatable="yes">View Closed Tasks</property>
+                    <signal name="clicked" handler="on_view_closed_toggled" swapped="no"/>
+                    <property name="valign">center</property>
                   </object>
-                  <packing>
-                    <property name="expand">False</property>
-                    <property name="homogeneous">True</property>
-                  </packing>
+                  <style>
+                    <class name="text-button"/>
+                  </style>
                 </child>
+              </object>
+            </child>
+            <child>
+              <object class="GtkButton" id="search">
+                <property name="visible">True</property>
+                <property name="sensitive">True</property>
+                <property name="tooltip_text" translatable="yes">Searching thing</property>
```

Comment:
Can you fix the tooltip_text. We might forget about it and user might be confused ;)